cwms-data-api/src/main/java/cwms/cda/data/dto/basin/Basin.java

```diff
@@ -0,0 +1,138 @@
+package cwms.cda.data.dto.basin;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.data.dto.CwmsDTOBase;
+import cwms.cda.data.dto.LocationIdentifier;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV1;
+
+@FormattableWith(contentType = Formats.JSONV1, formatter = JsonV1.class, aliases = {Formats.DEFAULT, Formats.JSON})
+@JsonDeserialize(builder = Basin.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public final class Basin implements CwmsDTOBase {
+    private final LocationIdentifier BASIN_ID;
```

Comment:
The names should be camel case, e.g. `basinId`. The chosen casing would be appropriate for constants, but these are just final variables for an immutable object.

Reply:
@MikeNeilson